manifests/proxydhcp.pp

```diff
@@ -33,7 +33,7 @@
   class { '::dhcp':
     dnsdomain   => $foreman_proxy::dhcp_option_domain,
     nameservers => $nameservers,
-    interfaces  => [$foreman_proxy::dhcp_interface],
+    interfaces  => [$foreman_proxy::dhcp_interface] + [$foreman_proxy::dhcp_additional_interfaces].flatten.delete_undef_values,
```

Comment:
I didn't know this was possible

Reply:
Yeah, Puppet 4 introduced some nice language features :)